doc/source/ray-core/handling-dependencies.rst

```diff
@@ -144,9 +144,11 @@ You can specify a runtime environment for your whole job, whether running a scri
 
 .. warning::
 
-    If using the Ray Jobs API (either the Python SDK or the CLI), specify the ``runtime_env`` argument in the ``submit_job`` call or the ``ray job submit``, not in the ``ray.init()`` call in the entrypoint script (in this example, ``my_ray_script.py``).
+    Specify the ``runtime_env`` argument in the ``submit_job`` call or the ``ray job submit`` ensures ensures the runtime environment is installed on the cluster before the entrypoint script is run.
 
-    This ensures the runtime environment is installed on the cluster before the entrypoint script is run.
+    If ``runtime_env`` is specified from ``ray.init(runtime_env=...)``, the runtime env is only applied to all children Tasks and Actors, not the entrypoint script (Driver) itself.
```

Comment:
From what I've seen, tasks, actors and drivers are not capitalized elsewhere in the doc. It would be great if you could double check and make it consistent

Reply:
This was recommended from @angelinalg. I will just stick to this it for now.